assets/training/vision/components/image_classification/spec.yaml

```diff
@@ -188,7 +188,7 @@ outputs:
 
 code: ../src
 
-environment: azureml:AzureML-AutoML-DNN-Vision-GPU:115
+environment: azureml://registries/azureml/environments/AzureML-AutoML-DNN-Vision-GPU/versions/117
```

Comment:
Why do we need to point to registry environment ? 
Wouldn't the CE from workspace env be picked up with the previous approach ?
If not, how did the runtime components already published by Gaurav/Daniel/Shubham work ?

Reply:
There are some recent changes in publishing pipelines, which is why these changes. 
Earlier, it used to work.